packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -36,23 +39,43 @@ export const Tab = React.forwardRef<HTMLAnchorElement, TabProps>(
       </>
     );
 
-    const linkClass = cx(tabsStyles.link, active ? tabsStyles.activeStyle : tabsStyles.notActive);
+    const linkClass = cx(clearStyles, tabsStyles.link, active ? tabsStyles.activeStyle : tabsStyles.notActive);
 
-    return (
+    if (href) {
       <div className={tabsStyles.item}>
         <a
-          // in case there is no href '#' is set in order to maintain a11y
-          href={href ? href : '#'}
+          href={href}
           className={linkClass}
           {...otherProps}
           onClick={onChangeTab}
           aria-label={otherProps['aria-label'] || selectors.components.Tab.title(label)}
           role="tab"
           aria-selected={active}
-          ref={ref}
+          // don't think we can avoid the type assertion here :(
+          // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+          ref={ref as React.ForwardedRef<HTMLAnchorElement>}
         >
           {content()}
         </a>
+      </div>;
+    }
+
+    return (
+      <div className={tabsStyles.item}>
+        <button
+          className={linkClass}
+          {...otherProps}
+          onClick={onChangeTab}
+          aria-label={otherProps['aria-label'] || selectors.components.Tab.title(label)}
+          role="tab"
+          type="button"
+          aria-selected={active}
+          // don't think we can avoid the type assertion here :(
+          // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+          ref={ref as React.ForwardedRef<HTMLButtonElement>}
```

Comment:
Most of these props are the same as for the anchor, should we extract them into a separate object, so they're not declared twice?